.expeditor/config.yml

```diff
@@ -110,6 +110,12 @@ subscriptions:
           post_commit: true
       - bash:.expeditor/purge_cdn.sh:
           post_commit: true
+  - workload: buildkite_build_canceled:{{agent_id}}:integration_test:*
+    actions:
+      - bash:.expeditor/integration_test.pipeline.sh destroy
+  - workload: buildkite_build_failed:{{agent_id}}:integration_test:*
+    actions:
+      - bash:.expeditor/integration_test.pipeline.sh destroy
```

Comment:
Unfortunately, you cannot pass parameters into `bash` actions. The script will need to detect that it needs to destroy based on the environment variables passed in. I would suggest the `EXPEDITOR_BUILD_STATE` environment variable, which would equal either `canceled` or `failed`.

Reply:
ah, ok sorry I didn't catch that!

I can work around that in the script using `EXPEDITOR_BUILD_STATE` as suggested.

On a side note can you verify that `EXPEDITOR_BUILD_NUMBER` would contain just the integer for the *failed | canceled* job?

Reply:
I've updated the shell script to override the action based on `EXPEDITOR_BUILD_STATE`:

https://github.com/chef/chef-server/pull/1874/files#diff-03877f02b41fcff024599b3a33aff44bR158-R159